ansible/slurm.yml

```diff
@@ -9,6 +9,15 @@
     - include_role:
         name:  mysql
 
+- name: Setup slurm-driven rebuild
+  hosts: rebuild
```

Comment:
```suggestion
  hosts: rebuild:!builder
```

Reply:
If this is turned on during build, we don't want the clouds.yml ending up in the image